website/cue/reference/releases/0.38.0.cue

```diff
@@ -0,0 +1,307 @@
+			description: """
+				Vector would panic when attempting to use a combination af `access_key_id` and
+				`assume_role` authentication with the AWS components. This error has now been
+				fixed.
+				"""
+			contributors: ["StephenWakely"]
+		},
+		{
+			type: "fix"
+			description: """
+				A `message` semantic meaning was added to the Splunk HEC source. This only applies to the `Vector` log namespace.
+				"""
+		},
+		{
+			type: "enhancement"
+			description: """
+				A new config field `missing_field_as` was added to the `databend` sink to specify the behavior when fields are missing. Previously the behavior was the same as setting this new configuration option to `ERROR`. The new default value is `NULL`.
+				"""
+			contributors: ["everpcpc"]
+		},
+		{
+			type: "feat"
+			description: """
+				The `clickhouse` sink now has a new configuration option, `insert_random_shard`, to tell Clickhouse to insert into a random shard (by setting `insert_distributed_one_random_shard`). See the Clickhouse [Distributed Table Engine docs](https://clickhouse.com/docs/en/engines/table-engines/special/distributed) for details.
+				"""
+			contributors: ["rguleryuz"]
+		},
+		{
+			type: "chore"
+			description: """
+				Previously the `datadog_agent` setting `parse_ddtags` parsed the tag string into an Object. It is now parsed into an Array of `key:value` strings, which matches the  behavior of the Datadog logs backend intake.
+				"""
+		},
+		{
+			type: "fix"
+			description: """
+				The `datadog_logs` sink was not re-constructing ddtags that may have been parsed upstream by the `datadog_agent` source's `parse_ddtags` setting. The sink log encoding was fixed to re-assemble the tags into a unified string that the Datadog logs intake expects.
+				"""
+		},
+		{
+			type: "feat"
+			description: """
+				The Elasticsearch sink is now able to use [external versioning for documents](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-index_.html#index-versioning). To use it set `bulk.version_type` to `external` and then set `bulk.version` to either some static value like `123` or use templating to use an actual field from the document `{{ my_document_field }}`.
+				"""
+			contributors: ["radimsuckr"]
+		},
+		{
+			type: "fix"
+			description: """
+				The `prometheus_exporter` sink is now able to correctly handle a mix of both incremental and
+				absolute valued gauges arriving for the same metric series.
+				"""
+			contributors: ["RussellRollins"]
+		},
+		{
+			type: "fix"
+			description: """
+				Previously, when the `auto_extract_timestamp` setting in the `splunk_hec_logs` Sink was enabled, the sink was attempting to remove the existing event timestamp. This would throw a warning that the timestamp type was invalid.
+
+				This has been fixed to correctly not attempt to remove the timestamp from the event if `auto_extract_timestamp` is enabled, since this setting indicates that Vector should let Splunk do that.
```

Comment:
```suggestion
				This has been fixed to leave the timestamp on the event if `auto_extract_timestamp` is enabled, since this setting indicates that Vector should let Splunk remove it.
```